### ISSUE TITLE
Docs: working through todoList tutorial after todos rather than starting from lb4 example does not create tests

### ISSUE THREAD

**Issue by cloudwheels** [strongloop/loopback-next#2051]
## Description / Steps to reproduce / Feature proposal
 - Work through the steps to build todoList on top of todos

## Current Behavior

- If the todoList tutorial is pulled from the repo / using `lb4 example` it will include tests.
- Tests are not automatically created if starting with the todo example and building todoList using `lb4 controller` etc 

## Expected Behavior

- Update docs: Remind users they may need to add tests 
- Make it easy to add existing tests (or use this opportunity for a tutorial-within-tutorial on building tests)

**Comment by bajtos** [comment]
I agree we should improve `lb4` tooling to help users to write tests for their code too. I think the following issue is pretty much covering your second point: _#885 "lb4 controller" should prepare test infrastructure_

Let's keep this issue focused on short-term improvements of our docs, i.e. _Update docs: Remind users they may need to add tests_.

@cloudwheels would you like to contribute this change yourself?

**Comment by cloudwheels** [comment]
Updated title to reflect your suggestions as basis for straightforward PR - yes can take this :)
Will get myself organised to contrib asap, may need a day or two if that's OK to balance things with the real world (mucho tiempo aqui!) but no longer...
